src/format.unit.ts

```diff
@@ -62,4 +62,38 @@ describe("format", () => {
 		form.Text = "label2";
 		expect(error.message).toBe("label2 must be formatted as #,###.");
 	});
+
+	test("lists", async () => {
```

Comment:
Can we improve the name of this test? If this fails, you would have to read through and understand the test code to figure out what the failure means.

`list items are formatted using the list property format`